efficientdet/keras/train.py

```diff
@@ -130,15 +130,15 @@ def main(_):
     tf.config.experimental_connect_to_cluster(tpu_cluster_resolver)
     tf.tpu.experimental.initialize_tpu_system(tpu_cluster_resolver)
     ds_strategy = tf.distribute.TPUStrategy(tpu_cluster_resolver)
-    logging.info("All devices: ", tf.config.list_logical_devices('TPU'))
+    logging.info('All devices: %s', tf.config.list_logical_devices('TPU'))
   elif FLAGS.strategy == 'gpus':
     ds_strategy = tf.distribute.MirroredStrategy()
-    logging.info("All devices: ", tf.config.list_physical_devices('GPU'))
+    logging.info('All devices: %s', tf.config.list_physical_devices('GPU'))
   else:
     if tf.config.list_physical_devices('GPU'):
```

Comment:
We need to either use ' or ". In this codebase, we prefer ' for consistency.

Reply:
Sorry for the mistake, could yapf fix them?

Reply:
Oh no worry at all. I just wanted to explain why this diff happens in this PR.

Unfortunately, the public yapf doesn't handle this issue, but the internal tool somehow automatically fix this. It is a small issue: you may pay attention to it later, but no need to worry about that if you happen to use ' and " mixed.

Reply:
BTW, I originally wrote "style=google" in FAQ, which was actually wrong.  This codebase uses "yapf" (then you don't need to add the extra SPLIT_BEFORE_EXPRESSION_AFTER_OPENING_PAREN.